comet_core/api_v0.py

```diff
@@ -143,8 +144,10 @@ def acceptrisk():
     """
     try:
         fingerprint = get_and_check_fingerprint()
+        record_metadata = hydrate_with_request_headers(request)
         get_db().ignore_event_fingerprint(fingerprint,
-                                          IgnoreFingerprintRecord.ACCEPT_RISK)
+                                          IgnoreFingerprintRecord.ACCEPT_RISK,
+                                          metadata=record_metadata)
```

Comment:
can we just call this record_metadata all over the code, instead of going from metadata to record_metadata at different part of the code?